mkt/tags/models.py

```diff
@@ -57,3 +56,23 @@ class AddonTag(ModelBase):
 
     class Meta:
         db_table = 'users_tags_addons'
+
+
+def attach_tags(objs, m2m_name=None):
+    """
+    Fetch tags from `objs` in one query and then attach them to a property on
+    each instance. The name of the property will be `m2m_name` + '_list'.
+
+    Assumes every instance in `objs` uses the same model, and needs `m2m_name`
+    to be set to the name of the m2m field between the model and Tag.
+    """
+    if m2m_name is None:
```

Comment:
I don't think we want to really enforce that (too much old code that would be inconsistent with that to make it a rule and not enough code using the decorator to really justify it, IMHO).

But you convinced me, I'll remove the check and the default value :)